<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThongKeDoanhThu</class>
 <widget class="QMainWindow" name="ThongKeDoanhThu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Báo cáo doanh thu</string>
  </property>
  <property name="styleSheet">
   <string>background-color: #F5F5F5; color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelStartDate">
        <property name="text">
         <string>Ngày bắt đầu:</string>
        </property>
        <property name="styleSheet">
         <string>color: black;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="startDateEdit">
        <property name="styleSheet">
         <string>background-color: #E6F3FF; color: black;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelEndDate">
        <property name="text">
         <string>Ngày kết thúc:</string>
        </property>
        <property name="styleSheet">
         <string>color: black;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="endDateEdit">
        <property name="styleSheet">
         <string>background-color: #E6F3FF; color: black;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generateReportButton">
        <property name="text">
         <string>Tạo báo cáo</string>
        </property>
        <property name="styleSheet">
         <string>background-color: #4CAF50; color: black;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="totalRevenueLabel">
      <property name="text">
       <string>Tổng doanh thu: 0 VNĐ</string>
      </property>
      <property name="styleSheet">
       <string>background-color: #FFF3E0; color: black;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="reportTable">
      <property name="styleSheet">
       <string>background-color: #FFFFFF; color: black; alternate-background-color: #F0F0F0;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <resources/>
  <connections/>
 </widget>
</ui>